package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@esm-bundle/react-dom",
-  "version": "17.0.1-fix.1",
+  "version": "17.0.2",
```

Comment:
`@esm-bundle/react` has 17.0.2 already so I also updated that here

Reply:
this might help